File: src/main/java/com/xeoneux/caravansary/repository/RoomRepository.java
package com.xeoneux.caravansary.repository;

import com.xeoneux.caravansary.entity.RoomEntity;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface RoomRepository extends CrudRepository<RoomEntity, Long> {
    RoomEntity findById(Long id);
}

File: src/main/kotlin/com/xeoneux/caravansary/repository/RoomRepository.kt
package com.xeoneux.caravansary.repository

import com.xeoneux.caravansary.entity.RoomEntity
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface RoomRepository : CrudRepository<RoomEntity, Long> {
    fun findById(id: Long?): RoomEntity
}
